homeassistant/components/webostv/config_flow.py

```diff
@@ -0,0 +1,203 @@
+"""Config flow to configure webostv component."""
+from __future__ import annotations
+
+import logging
+from urllib.parse import urlparse
+
+from aiopylgtv import PyLGTVPairException
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.components import ssdp
+from homeassistant.const import CONF_CLIENT_SECRET, CONF_HOST, CONF_NAME
+from homeassistant.core import callback
+from homeassistant.helpers import config_validation as cv
+
+from . import async_control_connect
+from .const import (
+    CONF_ON_ACTION,
+    CONF_SOURCES,
+    DEFAULT_NAME,
+    DOMAIN,
+    ON_ACTION_DOCS,
+    WEBOSTV_EXCEPTIONS,
+)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): cv.string,
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    },
+    extra=vol.ALLOW_EXTRA,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class FlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """WebosTV configuration flow."""
+
+    VERSION = 1
+
+    def __init__(self):
+        """Initialize workflow."""
+        self._host = None
+        self._name = None
+        self._uuid = None
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(config_entry):
+        """Get the options flow for this handler."""
+        return OptionsFlowHandler(config_entry)
+
+    async def async_step_import(self, import_info):
+        """Set the config entry up from yaml."""
+        self._host = import_info[CONF_HOST]
+        self._name = import_info.get(CONF_NAME) or import_info[CONF_HOST]
+        return await self.async_step_pairing()
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initialized by the user."""
+        errors = {}
+        if user_input is not None:
+            self._host = user_input[CONF_HOST]
+            self._name = user_input[CONF_NAME]
+            return await self.async_step_pairing()
+
+        return self.async_show_form(
+            step_id="user", data_schema=DATA_SCHEMA, errors=errors
+        )
+
+    async def async_check_configured_entry(self):
```

Comment:
```suggestion
    @callback
    def _async_check_configured_entry(self):
```

Nothing is being awaited here, this doesn't need to be a coro